src/models/ui.ts

```diff
@@ -1,4 +1,4 @@
-import { observable } from "mobx";
+import { observable, action } from "mobx";
```

Comment:
I see you added an import for `action` but I don't see it being used in this change.  

Just curious about that.

Reply:
Ah, I'll remove this import, thanks.
Generally, I think that actions should be used to set observable variables. But it seems to work that way, so I decided to keep things consistent. I guess quite a few things could be refactored in this repo, but I'm trying to avoid doing partial changes. If there's time left, I'll try to do more general cleanup and dependencies updates.